.github/scripts/sync_local.sh

```
@@ -0,0 +1,25 @@
+if [[ ! "$REPO_DIR" =~ ^[a-zA-Z0-9_-]+$ ]]; then
+    echo "Error: REPO_DIR must contain only alphanumeric characters, underscores, and hyphens" >&2
+    exit 1
+fi
+
+# determine root repo directory
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+
+# remove all files in the docs directory
+rm -rf $ROOT/../fiberDocs/docs/${REPO_DIR}/*
+
+for f in $(find -E . -type f -iregex '.*\.(md|png|jpe?g|gif|bmp|svg|webp)$' -not -path "./(fiberDocs)/*" -not -path "*/vendor/*" -not -path "*/.github/*" -not -path "*/.*"); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)